internal/bundleparams/parameters.go

```diff
@@ -1,4 +1,4 @@
-package commands
```

Comment:
nit: rename package to `bundle` ? This file here has some redundancy `bundleparams/parameters.go`.